ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/ict4dat/ICT4DatFragment.kt

```diff
@@ -23,7 +24,23 @@ class ICT4DatFragment : BaseNavigationFragment<ICT4DatViewModel, FragmentIct4dat
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return super.onCreateView(inflater, container, savedInstanceState)
+        val rootView = super.onCreateView(inflater, container, savedInstanceState)
+
+        binding.fragment = this
```

Comment:
may be we use `onViewCreated` method to set the binding, just an idea to save 8 lines which is occupied by `onCreateView`